src/components/forms/inputFieldOnly.vue

```diff
@@ -309,11 +334,17 @@ export default {
 		},
 		lostFocus: function() {
 			// run validators immediately on losing focus
+			if (typeof this.input === 'string') {
+				// trim string input values
+				this.input = this.input.trim();
+			}
 			this.isPristine = false;
-			this.runValidators();
+			if (this.readonly == false) {
```

Comment:
I'd recommend use of the identity check `===` over the use of equality `==`, for this and all other conditional checks.

Reply:
Sounds great. Made the changes and pushed to this PR